uniffi_bindgen/src/interface/mod.rs

```diff
@@ -317,17 +317,22 @@ impl ComponentInterface {
         self.is_name_used_as_error(&e.name) && (fielded || used_in_foreign_interface)
     }
 
-    /// Get details about all `Type::Custom` types
-    pub fn iter_custom_types(&self) -> impl Iterator<Item = (&String, &Type)> {
-        self.types.iter_known_types().filter_map(|t| match t {
-            Type::Custom { name, builtin, .. } => Some((name, &**builtin)),
-            _ => None,
-        })
+    /// Iterate over all known local types in the interface.
+    pub fn iter_types(&self) -> impl Iterator<Item = &Type> {
```

Comment:
Maybe this should change to `iter_local_types` too.  That means we need to update all the code that calls it, but that doesn't seem terrible to me.  The behavior is changing, so it's a good opportunity for us to double check that the code still makes sense.

Reply:
I was torn about this, so now you mentioned it it flipped me back over the edge :)